src/js/api/errors.ts

```diff
@@ -184,21 +193,21 @@ export class PlayerError {
     }
 }
 
-export function convertToPlayerError(key, code, error) {
+export function convertToPlayerError(key: ErrorKey | null, code: ErrorCode, error: Error | PlayerError): PlayerError {
```

Comment:
`key?:...` for optional params here

Reply:
same as above, unless `code` is not required? though then that would allow for empty errors.